packages/ui5-middleware-cfdestination/test/dest-in-env/ui5.yaml

```diff
@@ -0,0 +1,21 @@
+specVersion: "2.2"
+metadata:
+  name: ui5-app
+type: application
+framework:
+  name: OpenUI5
+  version: "1.115.1"
+  libraries:
+    - name: sap.m
+    - name: sap.ui.core
+    - name: sap.ui.layout
+    - name: themelib_sap_horizon
+server:
+  customMiddleware:
+  - name: ui5-middleware-cfdestination
+    afterMiddleware: compression
+    configuration:
+      debug: true
+      port: 1091
+      xsappJson: "xs-app.json"
+      destinations: "$env:xsapp_dest"
```

Comment:
I always thought of a general concept to load properties for the `ui5.yaml` from an env file to standardize this. Similar discussions we had for proxies and their username/password and we do similar things for the stringreplace middleware/task.

Cc @RandomByte @matz3 - something to think about for the evolution of the UI5 Tooling